TKeazirian.HTTPServer/Response/HeadResponse.cs

```diff
@@ -0,0 +1,25 @@
+namespace TKeazirian.HTTPServer.Response;
+
+using Handler;
+using Request;
+
+public class HeadResponse
+{
+    public Response BuildHeadResponse(Handler handler, Request request)
+    {
+        var getRequest = new Request(
+            "GET",
+            request.GetRequestPath(),
+            request.GetRequestHeaders(),
+            request.GetRequestBody()
+        );
+        var getResponse = handler.HandleResponse(getRequest);
+
+        var headResponse = new Response(
+            getResponse.GetStatusLine(),
+            getResponse.GetHeaders(),
+            ""
+        );
+        return headResponse;
+    }
+}
```

Comment:
This is an observation; typically in statically types languages, the use of `get` and `set` in names has a different meaning than how you are using it here.  In other words `getRequest` connotes "getting a request" and not an http GET verb.  For variables names, try to use nouns and reserve the use of verbs for the function names (like `request.GetRequestPath()`

Reply:
its fine, I see what is happening, its just ever so slightly confusing. Would not let it be a blocker.